packages/chargebee/src/action/index.tsx

```diff
@@ -32,12 +37,18 @@ const Action = ({
 
   let node;
   if (isReady) {
-    const handleClick = (event) => {
-      invoke(onClick, event);
-      if (!disabled) invoke(onCall, instanceRef.current, global.Chargebee);
+    const handleClick: React.MouseEventHandler<HTMLSpanElement> = (event) => {
+      onClick?.call?.(undefined, event);
+      if (!disabled) {
+        onCall?.call?.(
+          undefined,
+          instanceRef.current as ChargebeeInstance,
```

Comment:
The pattern doesn't sit right with me too but I think type assertion here is inevitable especially because the value here (`ref`) can be more than 1 type. We are already accounting for the null/undefined value, so it's safe to assume that the value is not null/undefined here